### ISSUE TITLE
Is there a way to ignore a property? 

### ISSUE THREAD

**Issue by YodasMyDad** [mrpmorris/blazor-validation#28]
I have created a custom FileUpload component for my project, which is essentially just a Blazor `InputFile` in a component and I've styled it and added a few other features onchange. 

It works absolutely fine / as expected on standard DataAnnotation validated forms. But as soon as I call `editContext.ValidateObjectTree()` on submit I get this error message?

```
blazor.server.js:19 [2021-01-21T15:58:52.610Z] Error: System.ArgumentException: The model must be a reference-typed object. (Parameter 'model')
   at Microsoft.AspNetCore.Components.Forms.FieldIdentifier..ctor(Object model, String fieldName)
   at PeterLeslieMorris.Blazor.Validation.Extensions.EditContextExtensions.NotifyPropertyChanged(EditContext editContext, Object instance, String propertyName)
   at PeterLeslieMorris.Blazor.Validation.Extensions.EditContextExtensions.ValidateObject(EditContext editContext, Object instance, HashSet`1 validatedObjects)
   at PeterLeslieMorris.Blazor.Validation.Extensions.EditContextExtensions.ValidateObject(EditContext editContext, Object instance, HashSet`1 validatedObjects)
   at PeterLeslieMorris.Blazor.Validation.Extensions.EditContextExtensions.ValidateObject(EditContext editContext, Object instance, HashSet`1 validatedObjects)
   at PeterLeslieMorris.Blazor.Validation.Extensions.EditContextExtensions.ValidateObject(EditContext editContext, Object instance, HashSet`1 validatedObjects)
   at PeterLeslieMorris.Blazor.Validation.Extensions.EditContextExtensions.ValidateObjectTree(EditContext editContext)
   at Gab.Web.Pages.Account.Manage.UserProfile.HandleSubmit(EditContext editContext) in C:\Users\lee\Documents\Projects\Gab\Src\Gab.Web\Pages\Account\Manage\UserProfile.razor.cs:line 52
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)
   at Microsoft.AspNetCore.Components.Rendering.RendererSynchronizationContext.ExecuteSynchronously(TaskCompletionSource`1 completion, SendOrPostCallback d, Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.Rendering.RendererSynchronizationContext.ExecuteBackground(WorkItem item)
```

Is there an option or attribute to get your package to ignore this property (Model/Property is either an `IBrowserFile` or `List<IBrowserFile>` just like for a standard `InputFile`)?

**Comment by mrpmorris** [comment]
Could you let me have a repro app to look at?

**Comment by mrpmorris** [comment]
Fixed by #33

**Comment by mrpmorris** [comment]
Fixed in 1.7
